lib/src/backend/invoker.dart

```diff
@@ -348,6 +352,14 @@ class Invoker {
         await _outstandingCallbacks.noOutstandingCallbacks;
         if (_timeoutTimer != null) _timeoutTimer.cancel();
 
+        if (liveTest.state.result != Result.success &&
+            _runCount < liveTest.test.metadata.retry + 1) {
+          _controller
+              .message(new Message.retry("Retry: ${liveTest.test.name}"));
```

Comment:
Are retry messages really worth a separate message type? We only have one for skips because we really want to print them in yellow text to match skipped tests.